src/ctim/schemas/actor.cljc

```diff
@@ -23,9 +23,9 @@
   c/described-entity-entries
   (f/required-entries
    (f/entry :type ActorTypeIdentifier)
-   (f/entry :valid_time c/ValidTime)
-   (f/entry :actor_type v/ThreatActorType))
+   (f/entry :valid_time c/ValidTime))
   (f/optional-entries
+   (f/entry :actor_type v/ThreatActorType)
```

Comment:
What about changing it to a collection as I suggested in this https://github.com/threatgrid/ctim/issues/381#issuecomment-1314889560?

Reply:
if there are any in prod today, we will have to migrate it. But yes that would be nice (and that's an easy one to migrate)